main.go

```diff
@@ -34,7 +34,7 @@ func main() {
 	}
 
 	cfg := packages.Config{
-		Mode:  packages.NeedSyntax | packages.NeedName | packages.NeedFiles | packages.NeedTypes | packages.NeedTypesInfo | packages.NeedDeps,
+		Mode:  packages.NeedSyntax | packages.NeedName | packages.NeedFiles | packages.NeedTypes | packages.NeedTypesInfo,
 		Tests: *includeTests,
```

Comment:
I didn't mean to remove this but which case do we need it for?  I could imagine it being expensive.

Reply:
Remove it from `forbidigo/forbidigo_test.go` and parsing will fail with:
```
2022/12/05 09:06:46 internal error: package "fmt" without types was imported from "_/tmp/TestForbiddenIdentifiersit_finds_forbidden__renamed_identifiers1395216622/001"
```